### PR TITLE
Modern Chainmail

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Updates chainmail to use steel, adds kevlar gambeson variants."

#### Purpose of change
Iron chainmail existed but steel is more common and more likely to be crafted by a survivor using existing steel wire/scrap. 

#### Describe the solution
Replace all chainmail materials using "iron" with "steel". Lower thickness of material to approximate current defensive values. Adds kevlar gambeson versions of the chainmail suits. Adds an XL faraday chainmail armor.

#### Describe alternatives you've considered
Changing all iron chainmail to use iron or scrap metal based wire.

#### Testing
Debug spawned all added items, successfully wore. Tested recipes.

#### Additional context
Updated based on comments in #52731. Weapon rebalancing is out of scope for this PR.

### CODE REVIEW

**Comment by SunshineDistillery** [comment]
Historical chainmail didn't use drawn wire, which is why it was so labor intensive. They had to pound out all those little rods and shape them into rings. You might want to reduce the construction times as well.

**Comment by Drew4484** [comment]
> Historical chainmail didn't use drawn wire, which is why it was so labor intensive. They had to pound out all those little rods and shape them into rings. You might want to reduce the construction times as well.

I considered reducing crafting times and adding a washer variant construction option, however adjusting the crafting would require a new way to make iron wire for rings, for washers an entirely new item would have to be added. I'm not adverse to either, but wanted to limit the non-equipment changes as much as possible.

**Comment by anoobindisguise** [comment]
#52731 I should note that Kevin does not want these changes to happen without first nerfing chainmail. I tried to do the exact same changes myself.
Double posting this because it is highly relevant to this case.

**Comment by kevingranade** [comment]
> Historical chainmail didn't use drawn wire

Source? That's contrary to my understanding.

**Comment by Wishbringer** [comment]
Quick google search (keywords: chainmail ringlets) reveals:
Chainmail was made by linking together thousands of small metal ringlets.
These ringlets were forged from tiny strips of iron and later steel, with the rings joined in a specific pattern and then closed by pressing, welding or riveting.

**Comment by SunshineDistillery** [comment]
https://www.academia.edu/11972089/The_history_and_evolution_of_wiredrawing_techniques

Looks like I was wrong. They did have dies for wires back then.

File: data/json/items/armor/suits_protection.json [hunk 1]
```diff
@@ -458,14 +458,22 @@
...
-    "material": [ "iron", "cotton" ],
+    "material": [ "steel", "cotton" ],
     "symbol": "[",
     "looks_like": "armor_blarmor",
     "color": "light_red",
     "warmth": 30,
     "material_thickness": 6,
     "flags": [ "VARSIZE", "STURDY", "OUTER" ],
-    "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ] } ]
+    "armor": [ { "encumbrance": 10, "coverage": 100, "covers": [ "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ]
```
Comment (kevingranade) [inline data/json/items/armor/suits_protection.json:468]:
```suggestion
    "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ]
```

File: data/json/items/armor/suits_protection.json [hunk 2]
```diff
@@ -486,14 +502,24 @@
...
+    "material": [ "steel", "cotton" ],
     "symbol": "[",
     "looks_like": "touring_suit",
     "color": "light_red",
     "warmth": 30,
-    "material_thickness": 6,
+    "material_thickness": 5,
     "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT", "OUTER" ],
-    "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "torso", "head", "arm_l", "arm_r", "leg_l", "leg_r" ] } ]
+    "armor": [ { "encumbrance": 10, "coverage": 100, "covers": [ "head", "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ]
```
Comment (kevingranade) [inline data/json/items/armor/suits_protection.json:512]:
```suggestion
    "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "head", "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ]
```

File: data/json/items/armor/suits_protection.json [hunk 3]
```diff
@@ -486,14 +502,24 @@
...
+  },
+  {
+    "id": "kevlar_chainmail_suit",
+    "type": "ARMOR",
+    "name": { "str": "kevlar chainmail armor" },
+    "description": "A fully customized chainmail suit.  The shirt, arms, and leggings have been modified with straps and combined with a kevlar gambeson to deal with uneven weight distribution, cold environments and to allow them to be used separately.",
+    "copy-from": "chainmail_suit",
+    "material": [ { "type": "steel", "portion": 5 }, { "type": "kevlar_layered", "portion": 3 }, { "type": "kevlar", "portion": 2 } ],
+    "armor": [ { "encumbrance": 14, "coverage": 100, "covers": [ "head", "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ],
```
Comment (kevingranade) [inline data/json/items/armor/suits_protection.json:521]:
```suggestion
    "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "head", "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ],
```

File: data/json/recipes/armor/head.json [hunk 4]
```diff
@@ -1829,6 +1829,12 @@
     "components": [ [ [ "link_sheet", 2 ] ], [ [ "chain_link", 50 ] ], [ [ "wire", 1 ] ], [ [ "rag", 4 ] ] ],
     "proficiencies": [ { "proficiency": "prof_chain_armour" } ]
   },
+  {
+    "result": "chainmail_hood",
+    "type": "recipe",
+    "copy-from": "chainmail_hood",
+    "components": [ [ [ "link_sheet", 2 ] ], [ [ "chain_link", 50 ] ], [ [ "wire", 1 ] ], [ [ "rag", 4 ] ] ]
+  },
```
Comment (nornagon) [inline data/json/recipes/armor/head.json:1837]: I'm a bit confused, what is this supposed to achieve? This is the same components as the recipe it's copied from. Can this be extra recipe be deleted?

Comment (Drew4484) [reply data/json/recipes/armor/head.json:1837]: Yes, this is a duplicate that can go. I'm not at my desktop right now to fix it, but can get to it later.

**Pull request merged**